common/src/main/scala/package.scala

```diff
@@ -61,8 +59,22 @@ package object eoi {
       }
     }
 
-    def formatPercentage: String = NF.getPercentInstance(UK).format(in)
-    def formatSterling: String = NF.getCurrencyInstance(UK).format(in)
+    def formatSterling: String = formatSterling(false)
+
+    def formatSterling(alwaysShowPence: Boolean): String =
```

Comment:
having a single function with default argument equal to `false` instead of an two separate functions would prevent users having to look at the implementation of `formatSterling` to see what the parameterless version uses

Reply:
true, but I wanted it to work without parenthesis, e.g. - `someval.formatSterling`